draft-ietf-quic-http.md

```diff
@@ -1229,10 +1229,13 @@ A server MAY accept 0-RTT and subsequently provide different settings in its
 SETTINGS frame. If 0-RTT data is accepted by the server, its SETTINGS frame MUST
 NOT reduce any limits or alter any values that might be violated by the client
 with its 0-RTT data.  The server MUST include all settings which differ from
-their default values.  If a server accepts 0-RTT, but then sends a SETTINGS
-frame which reduces a setting the client understands or omits a value that was
-previously specified to have a non-default value, this MUST be treated as a
-connection error of type HTTP_SETTINGS_ERROR.
+their default values.  If a server accepts 0-RTT but then sends a SETTINGS
+frame which reduces setting the client understands, this MUST be treated as a
```

Comment:
An interesting consequence of the "reduces" clause here is that you can't define a "supports" setting with 0=supports and 1=doesn't support.

Reply:
reduces "a" setting.  #2972 switched to using the term "compatible" up at lines 1215-1216, including a definition of the term; perhaps using "incompatible" instead of "reduces" here as well would help?

Reply:
Thanks for catching the missing "a" in "reduces a setting" (although it's moot now that I've reworded this).

I like using the term "compatible" instead of reduces, since it avoids this odd edge case and imo is more specific.